package.json

```diff
@@ -48,7 +48,8 @@
         "source-map-support": "^0.5.21",
         "typescript": "next",
         "w3c-xmlserializer": "^2.0.0",
-        "yargs": "^17.1.1"
+        "yargs": "^17.1.1",
+        "csstype": "^3.0.2"
```

Comment:
> I add `csstype` to `package.json` to pass the CI test as it's a dependency of `@types/styled-components`. I'm not sure if I'm allowed to do so, but it's the only way I can get the CI test passed.
> 
> https://github.com/DefinitelyTyped/DefinitelyTyped/blob/b0d22c4a76553976ff51fdc9068c341966dd1506/types/styled-components/package.json#L3-L5
> 
> 
> https://github.com/DefinitelyTyped/DefinitelyTyped/blob/b0d22c4a76553976ff51fdc9068c341966dd1506/types/styled-components/index.d.ts#L23-L33
> 
> 
> If it's not allowed, I can revert it.

@xsjcTony please revert, you shouldn't add a DT package's (styled-components) dependency to this top-level `package.json`. I understand CI was failing, so if when you remove this CI goes back to failing, I can help investigate why that's happening and find another way to fix it instead.

Reply:
I'll revert it, the cause of CI fail is because it's not going to install the dependency of the DT package (styled-components in this case) hence related Type Errors are raised.